src/basic/special.h

```diff
@@ -97,6 +97,7 @@
 #define SPECIAL_PCRFS_ROOT_SERVICE "systemd-pcrfs-root.service"
 #define SPECIAL_VALIDATEFS_SERVICE "systemd-validatefs@.service"
 #define SPECIAL_HIBERNATE_RESUME_SERVICE "systemd-hibernate-resume.service"
+#define SPECIAL_DMCLONE_SERVICE "systemd-dmclone@.service"
```

Comment:
hmm, not too fond of the "dm" in the name, we don#t name cryptsetup/veritysetup/integritysetup with the "dm" in the name.

i think the service should be called clonesetup@.service, to follow the current naming scheme